src/core/lib/security/credentials/s2a/grpc_s2a_credentials_options.h

```diff
@@ -30,37 +31,46 @@ struct grpc_s2a_credentials_options {
  public:
   ~grpc_s2a_credentials_options();
 
-  /** Setter methods. They do not take ownership of the arguments, nor do they
-   * remove a duplicate ciphersuite or target service account from the
-   * appropriate vector, if a duplicate exists. **/
+  /** Getters for member fields. **/
+  const std::string handshaker_service_url() const {
+    return handshaker_service_url_;
+  }
+  const std::vector<uint16_t>& supported_ciphersuites() const {
+    return supported_ciphersuites_;
+  }
+  const std::vector<std::string>& target_service_account_list() const {
+    return target_service_account_list_;
+  }
+
   /** The setter method for |handshaker_service_url_|. This does not take
    *  ownership of the argument. **/
```

Comment:
Since we pass by value, I do not think the comment about ownership holds. Same for other APIs below.

Reply:
Removed this comment in next commit.